src/Patch/Document.js

```diff
@@ -0,0 +1,67 @@
+import Native from './Native';
+import CustomElementInternals from '../CustomElementInternals';
+import * as Utilities from '../Utilities';
+
+import PatchParentNode from './Interface/ParentNode';
+
+/**
+ * @param {!CustomElementInternals} internals
+ */
+export default function(internals) {
+  Utilities.setPropertyUnchecked(Document.prototype, 'createElement',
+    /**
+     * @this {Document}
+     * @param {string} localName
+     * @return {!Element}
+     */
+    function(localName) {
+      const definition = internals.localNameToDefinition(localName);
+      if (definition) {
```

Comment:
This should be fixed. I mark documents that are meant to be associated with the registry with a flag and only customize elements if they're created in the context of one of those documents.